orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -339,6 +340,15 @@ def _anonymise_study_instances(
         with zipped_study.open(file_info) as file:
             logger.debug("Reading file {}", file)
             dataset = dcmread(file)
+
+            if dataset.SeriesInstanceUID in series_to_skip:
+                logger.warning(
+                    "Skipping series {} for study {} due to too few instances",
+                    dataset.SeriesInstanceUID,
+                    study_info,
+                )
+                continue
```

Comment:
Oh we should also add this to the `skipped_instance_counts` and change this to debug to follow what we've done for other ways of skipping

Reply:
ah yes! I've added the counts